spec/features/group_browsing_spec.rb

```diff
@@ -27,13 +27,22 @@
     expect(page).to have_link('A Leaf Node')
   end
 
-  scenario 'A team with subteams' do
+  scenario 'A team with people and subteams with people' do
     current_group = team
     add_people_to_group(names, current_group)
+    add_people_to_group(names, subteam)
     visit group_path(current_group)
 
     expect(page).to have_text("Teams within #{ current_group.name }")
     expect(page).to have_link("View all people in #{ current_group.name }")
+    expect(page).to have_text("#{subteam.completion_score}% of profile information completed")
+  end
+
+  scenario 'A team and subteams without people' do
+    current_group = team
+    visit group_path(current_group)
+
+    expect(page).to have_text("0% of profile information completed")
```

Comment:
if there is no people at all in any subteam, maybe we shouldn't display this. Because 0% would suggest profiles are very incomplete when in fact they're missing altogether.